src/components/views/rooms/MessageComposer.js

```diff
@@ -116,15 +116,16 @@ export default class MessageComposer extends React.Component {
         }
 
         Modal.createDialog(QuestionDialog, {
-            title: "Upload Files",
+            title: _t('Upload Files'),
             description: (
                 <div>
-                    <p>Are you sure you want upload the following files?</p>
+                    <p>{ _t('Are you sure you want upload the following files?') }</p>
                     <ul style={{listStyle: 'none', textAlign: 'left'}}>
                         {fileList}
                     </ul>
                 </div>
             ),
+            button: "OK",
```

Comment:
Solved, but unfortunately, the OK must be in the caller, since in the basedialog the language wasn't yet loaded...